src/index.js

```diff
@@ -171,7 +193,11 @@ const renderTodo = (todo) => {
           </p>
           <div class="clearfix todo-item-footer">
               <h6 class="float-left">Project: </h6>
-              <span cclass="float-left">${todo.name}</span>
+              <span cclass="float-left">${projectName}</span>
+          </div>
+           <div class="clearfix todo-item-footer">
```

Comment:
is there are real reason we chose not to create forms with the `form` tag?
- there are advantages to that. we could have used the submit event so either button click or enter will trigger a method call.
- we can call form rest on it to wipe out the fields when we have submitted the form.

Reply:
I see the advantage you mentioned. We did not think in the direction of a form to be submitted to any backend. Besides we don't need to reset the entire form all the time only a few fileds require a reset. This allows you to continue adding tasks to a project without having to select project all the time except you want a change in project. I think thats a better user experience